specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-11-01-preview/examples/CreateOrUpdateArcSqlServerAvailabilityGroup.json

```diff
@@ -0,0 +1,259 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "sqlServerInstanceName": "testSqlServer_INST1",
+    "availabilityGroupName": "testAG",
+    "api-version": "2023-11-01-preview",
+    "sqlServerAvailabilityGroupResource": {
+      "location": "southeastasia",
+      "tags": {
+        "mytag": "myval"
+      },
+      "properties": {
+        "availabilityGroupId": "00000000-1111-2222-3333-444444444444",
+        "serverName": "testSqlServer",
+        "instanceName": "INST1",
+        "collectionTimestamp": "2022-05-05T16:26:33.883Z",
+        "info": {
+          "failureConditionLevel": 3,
+          "healthCheckTimeout": 30000,
+          "automatedBackupPreferenceDescription": "SECONDARY",
+          "version": 0,
+          "basicFeatures": false,
+          "dtcSupport": false,
+          "dbFailover": true,
+          "isDistributed": false,
+          "clusterTypeDescription": "WSFC",
+          "requiredSynchronizedSecondariesToCommit": 0,
+          "isContained": false,
+          "primaryReplica": "testSqlServer\\INST1",
+          "primaryRecoveryHealthDescription": "ONLINE",
+          "secondaryRecoveryHealthDescription": null,
+          "synchronizationHealthDescription": "HEALTHY"
+        },
+        "replicas": {
+          "value": [
+            {
+              "replicaId": "00000000-1111-2222-3333-444444444444",
+              "replicaName": "testSqlServer\\INST1",
+              "configure": {
+                "endpointUrl": "TCP://mytest60-0.mytest60-svc:5022",
+                "availabilityModeDescription": "SYNCHRONOUS_COMMIT",
```

Comment:
>SYNCHRONOUS_COMMIT [](http://example.com/codeflow?start=48&length=18)

Why is it all caps?

Is this really a description, or is it just an enum with values?

You should declare all the enum values.